scripts/tests/chiptest/test_definition.py

```diff
@@ -100,18 +109,18 @@ def wait(self, timeout=None):
                 while self.stopped:
                     self.cv_stopped.wait()
 
-    def __startServer(self, runner, command, options):
+    def __startServer(self, runner, command):
         app_cmd = command + ['--interface-id', str(-1)]
 
-        if not options:
+        if not self.options:
             logging.debug('Executing application under test with default args')
         else:
             logging.debug('Executing application under test with the following args:')
-            for key, value in options.items():
+            for key, value in self.options.items():
                 logging.debug('   %s: %s' % (key, value))
                 app_cmd = app_cmd + [key, value]
                 if key == '--KVS':
-                    self.kvs = value
+                    self.kvsList.add(value)
```

Comment:
nit: It would be nice to not add duplicates to the list

Reply:
`self.kvsList` is a `set`, so it handles duplicate bits itself.

Reply:
Yeah, I had initially used a list and did a manual check here before adding but Boris pointed me to using a set